optimizely/helpers/enums.py

```diff
@@ -221,3 +221,9 @@ class OdpManagerConfig:
     """ODP Manager configs."""
     KEY_FOR_USER_ID: Final = 'fs_user_id'
     EVENT_TYPE: Final = 'fullstack'
+
+
+class OdpSegmentsCacheConfig:
+    """ODP Segment Cache configs."""
+    DEFAULT_CAPACITY: Final = 10_000
+    DEFAULT_TIMEOUT_SECS: Final = 10
```

Comment:
default timeout should be 10mins = 600secs

Reply:
Do we have a test validating the default settings?

Reply:
thx, adding